### ISSUE TITLE
can not send up to 25 message

### ISSUE THREAD

**Issue by itsosmx** [fireship-io/react-firebase-chat#16]
after send 25 message the new messages stored on the database but the problem is there is no new divs created or anything happen in the chat box and there is no errors

**Comment by CharlieGroves** [comment]
I am having the same issue

**Comment by CharlieGroves** [comment]
Use .limitToLast(25) instead of .limit(25) if you are having this issue.

Someone in the comments of the video figured that out. Hope you see this!

**Comment by ghost** [comment]
yea it is because there is limit in the video as well that you cannot send more than 25 messages at once

**Comment by karljohan122** [comment]
Well an easy fix for that is just to remove the .limit(25) from the query constant. I don't get how nobody figured this out?

**Comment by CharlieGroves** [comment]
Karl, the issue was the app only showing the oldest 25 messages when using .limit(25) whereas it shows the 25 most recent messages using .limitToLast(25). Of course you can change 25 to any number but that wasn't the issue, the issue was it showing older messages rather than the latest messages.

**Comment by karljohan122** [comment]
> Karl, the issue was the app only showing the oldest 25 messages when using .limit(25) whereas it shows the 25 most recent messages using .limitToLast(25). Of course you can change 25 to any number but that wasn't the issue, the issue was it showing older messages rather than the latest messages.

No, yes I get that it would show only the older 25 messages but I don't get why you would even limit them in the first place? It still stores the messages in Firestore but won't display them, just taking up unnecessary space in Firestore. Is this some part of the design that I just don't get? I was having the same issue with the chat just displaying only the older 25 messages so I just decided to remove the limit.
